@mike-ainsel 
```
--> Scenario: 'cloud-epc-delegated'
--> Action: 'lint'
--> Executing Yamllint on files found in /home/gitlab-runner/builds/U6XQT3tv/0/lean-delivery/ansible-role-solr-hybris-config/...
    /home/gitlab-runner/builds/U6XQT3tv/0/lean-delivery/ansible-role-solr-hybris-config/defaults/main.yml
      47:1      error    too many blank lines (1 > 0)  (empty-lines)```

--- a/.gitlab-ci.yml
+++ b/.gitlab-ci.yml
@@ -37,7 +37,7 @@ AWS solr-hybris-config:
     AWS_REGION: us-east-1
   stage: deployment test
   script:
-    - molecule --debug test -s cloud-aws-direct
+    - molecule test -s cloud-aws-direct
   tags:
     - aws
 
@@ -46,7 +46,7 @@ AWS Windows solr-hybris-config:
     AWS_REGION: us-east-1
   stage: deployment test
   script:
-    - molecule --debug test -s cloud-aws-windows
+    - molecule test -s cloud-aws-windows
   tags:
     - aws
 

--- a/.gitlab-ci.yml
+++ b/.gitlab-ci.yml
@@ -37,7 +37,7 @@ AWS solr-hybris-config:
     AWS_REGION: us-east-1
   stage: deployment test
   script:
-    - molecule test -s cloud-aws-direct
+    - molecule --debug test -s cloud-aws-direct
   tags:
     - aws
 
@@ -46,7 +46,7 @@ AWS Windows solr-hybris-config:
     AWS_REGION: us-east-1
   stage: deployment test
   script:
-    - molecule test -s cloud-aws-windows
+    - molecule --debug test -s cloud-aws-windows
   tags:
     - aws
 
@@ -55,6 +55,6 @@ EPC solr-hybris-config:
     EPC_REGION: EPAM-BY2
   stage: deployment test
   script:
-    - molecule test -s cloud-epc-delegated
+    - molecule --debug test -s cloud-epc-delegated
   tags:
     - delegated

--- a/defaults/main.yml
+++ b/defaults/main.yml
@@ -44,4 +44,3 @@ solr_maven_libs_list:
   - 'jcseg-elasticsearch'
   - 'jcseg-server'
 solr_maven_libs_dest: '{{ solr_dest_path }}/server/solr-webapp/webapp/WEB-INF/lib'
-

--- a/molecule/cloud-aws-direct/molecule.yml
+++ b/molecule/cloud-aws-direct/molecule.yml
@@ -12,10 +12,11 @@ lint:
 platforms:
   - name: test-aws-centos7-solr-hybris-config
     image: ami-9887c6e7
+    platform: centos7
     instance_type: m5.large
-    profile_name: epmldi-instance-profile
     region: us-east-1
-    vpc_subnet_id: subnet-05a2ef2b767afec50
+    vpc_subnet_id: subnet-0f2b9cd66faea38af
+    profile_name: epmldi-instance-profile
     assign_public_ip: False
     spot_price: 0.04
     security_group_name:
@@ -28,9 +29,10 @@ platforms:
 
   - name: test-aws-ubuntu16-solr-hybris-config
     image: ami-09677e0a6b14905b0
+    platform: ubuntu16
     instance_type: m5.large
     region: us-east-1
-    vpc_subnet_id: subnet-05a2ef2b767afec50
+    vpc_subnet_id: subnet-0f2b9cd66faea38af
     assign_public_ip: False
     security_group_name:
       - default
@@ -43,10 +45,11 @@ platforms:
 
   - name: test-aws-ubuntu18-solr-hybris-config
     image: ami-012fd5eb46f56731f
+    platform: ubuntu18
     instance_type: m5.large
     profile_name: epmldi-instance-profile
     region: us-east-1
-    vpc_subnet_id: subnet-05a2ef2b767afec50
+    vpc_subnet_id: subnet-0f2b9cd66faea38af
     assign_public_ip: False
     security_group_name:
       - default
@@ -59,9 +62,10 @@ platforms:
 
   - name: test-aws-Debian9-solr-hybris-config
     image: ami-003f19e0e687de1cd
+    platform: debian9
     instance_type: m5.large
     region: us-east-1
-    vpc_subnet_id: subnet-05a2ef2b767afec50
+    vpc_subnet_id: subnet-0f2b9cd66faea38af
     assign_public_ip: False
     spot_price: 0.04
     security_group_name:

--- a/molecule/cloud-aws-windows/molecule.yml
+++ b/molecule/cloud-aws-windows/molecule.yml
@@ -12,10 +12,11 @@ lint:
 platforms:
 
   - name: test-aws-Windows2016-solr-hybris-config
-    aws_image_id: ami-00a20f508263efd30
-    aws_region: us-east-1
+    image: ami-00a20f508263efd30
+    platform: windows2016core
+    region: us-east-1
     instance_type: m5.large
-    vpc_subnet_id: subnet-01be2149
+    vpc_subnet_id: subnet-314c223d
     assign_public_ip: True
     spot_price: 0.14
     aws_sec_group:
@@ -42,8 +43,6 @@ provisioner:
     destroy: ../resources/provisioning/WINRM/destroy.yml
   lint:
     name: ansible-lint
-  env:
-    ANSIBLE_LIBRARY: ${ANSIBLE_LIBRARY}
 scenario:
   name: cloud-aws-windows
 verifier:

